examples/hosts/app-integration/external-data/src/model/appModel.ts

```diff
@@ -13,7 +14,15 @@ import type { IAppModel, IAppModelEvents, ITaskList } from "../modelInterfaces";
  * responsibilities and functionality.
  */
 export class AppModel extends TypedEventEmitter<IAppModelEvents> implements IAppModel {
-    public constructor(public readonly taskList: ITaskList, container: IContainer) {
+    public constructor(
+        public readonly taskList: ITaskList,
+        container: IContainer,
+        private readonly runtime: IContainerRuntime ) {
         super();
+        this.runtime = runtime;
+    }
+
+    private readonly debugSendCustomSignal = (): void => {
+        this.runtime.submitSignal('debugSignal', {message: "externalDataChanged"});
```

Comment:
Please add a comment with a little detail about how this signal works (e.g. what we expect the server will do in response).

Also a nit here on formatting (I thought we had an eslint rule that would catch this?)
```suggestion
        this.runtime.submitSignal("debugSignal", {message: "externalDataChanged"});
```